spring-security/src/test/java/com/sap/cloud/security/spring/token/authentication/ReactiveHybridJwtDecoderTest.java

```diff
@@ -15,27 +16,28 @@
 
 import static com.sap.cloud.security.config.Service.IAS;
 import static com.sap.cloud.security.config.Service.XSUAA;
+import static org.junit.jupiter.api.Assertions.*;
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.Mockito.when;
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertThrows;
 
-public class ReactiveHybridJwtDecoderTest {
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
```

Comment:
if you use per class lifecycle what do you share among the test instances in this test class?